spyrmsd/graph.py

```diff
@@ -1,42 +1,97 @@
+from spyrmsd import constants
+
+import numpy as np
+import os
+
+available_backends = []
+
 try:
     from spyrmsd.graphs.gt import (
-        cycle,
-        graph_from_adjacency_matrix,
-        lattice,
-        match_graphs,
-        num_edges,
-        num_vertices,
-        vertex_property,
+        cycle as gt_cycle,
+        graph_from_adjacency_matrix as gt_graph_from_adjacency_matrix,
+        lattice as gt_lattice,
+        match_graphs as gt_match_graphs,
+        num_edges as gt_num_edges,
+        num_vertices as gt_num_vertices,
+        vertex_property as gt_vertex_property,
     )
-
+    available_backends.append("graph-tool")
 except ImportError:
-    try:
-        from spyrmsd.graphs.nx import (
-            cycle,
-            graph_from_adjacency_matrix,
-            lattice,
-            match_graphs,
-            num_edges,
-            num_vertices,
-            vertex_property,
-        )
-    except ImportError:
-        raise ImportError("graph_tool or NetworkX libraries not found.")
-
-__all__ = [
-    "graph_from_adjacency_matrix",
-    "match_graphs",
-    "vertex_property",
-    "num_vertices",
-    "num_edges",
-    "lattice",
-    "cycle",
-    "adjacency_matrix_from_atomic_coordinates",
-]
-
-import numpy as np
-
-from spyrmsd import constants
+    pass
+    
+try:
+   from spyrmsd.graphs.nx import (
+        cycle as nx_cycle,
+        graph_from_adjacency_matrix as nx_graph_from_adjacency_matrix,
+        lattice as nx_lattice,
+        match_graphs as nx_match_graphs,
+        num_edges as nx_num_edges,
+        num_vertices as nx_num_vertices,
+        vertex_property as nx_vertex_property,
+    )
+   available_backends.append("networkx")
+except ImportError:
+   pass
+        
+def get_available_backends():
+    return available_backends 
```

Comment:
```suggestion
def available_backends():
    return _available_backends 
```
Mainly because I have PSTD for all `set_*` and `get_*` methods that were so popular when I learned OOP in C++...